### ISSUE TITLE
Add a toggle to show and hide the text input to the ManualInput.TextField

### ISSUE THREAD

**Issue by weiluntong** [Arian04/android-hid-client#51]
**Is your feature request related to a problem? Please describe.**
I'm so happy I found this program. This program has allowed me to use Keepass2Android (K2A) on computers that don't contain my password file (or even for computers that I login to).

It works like a charm, but the first thing I noticed is that the text field is visible. Since the input is sent through USB, I'm not too worried about the data being intercepted (barring keyloggers on my phone or the device I'm sending input to), but it would be easy for someone to look over my shoulder and see what's in the box. Also, with the vanilla TextField, some keyboards remember text that has been typed. When I autofill the TextField using K2A, that's not a worry, but if I happen to manually type into SwiftKey, it'll remember it there. I wouldn't want any sensitive information to be suggested on SwiftKey just in case someone is using my phone.

Sorry, not concise. Here:
TL;DR I want to toggle situations to hide manual input TextField and prevent keyboard from remembering input

**Describe the solution you'd like**
Add the usual Toggling Eye icon that will hide the text in the TextField and change the KeyboardType to one that prevents remembering keyboard input.

**Describe alternatives you've considered**
Uh... use a privacy screen protector and a dumber software keyboard I guess?

**Comment by Arian04** [comment]
This feature was added in PR #52 which was merged on 12/7/2025.
